src/ODBC/OdbcNativeMetadataProvider.php

```diff
@@ -0,0 +1,234 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractor\Adapter\ODBC;
+
+use Keboola\Datatype\Definition\GenericStorage;
+use Keboola\DbExtractor\Adapter\Metadata\MetadataProvider;
+use Keboola\DbExtractor\TableResultFormat\Metadata\Builder\ColumnBuilder;
+use Keboola\DbExtractor\TableResultFormat\Metadata\Builder\MetadataBuilder;
+use Keboola\DbExtractor\TableResultFormat\Metadata\Builder\TableBuilder;
+use Keboola\DbExtractor\TableResultFormat\Metadata\ValueObject\Table;
+use Keboola\DbExtractor\TableResultFormat\Metadata\ValueObject\TableCollection;
+use Keboola\DbExtractorConfig\Configuration\ValueObject\InputTable;
+
+/**
+ * Returns table and column metadata using the PHP functions: "odbc_tables" and "odbc_columns".
+ */
+class OdbcNativeMetadataProvider implements MetadataProvider
```

Comment:
`MetadataProvider` vracia v db-extractoroch zoznam tabuliek a stlpcov.
Vacsinou sa to robi ako `SELECT` z nejakej `information_schema`.

V ODBC PHP existuju funkcie `odbc_tables` a `odbc_columns`, ktore to vratia unifikovane pre kazdy driver.
Niektore drivery to nepodporuju, ale Hive ano.



Uz sa to tam pouziva, cez DiBi:
https://github.com/keboola/db-extractor-hive/blob/master/src/Connection/HiveOdbcReflector.php

Kedze v `db-extractor-common`:14+, na ktory to updatujeme musi byt definovany `MetadataProvider` -> tak som vytvoril tento `OdbcNativeMetadataProvider`.

A kedze je to univerzalne pre viacero driverov, tak to nedavam ku Hive, ale tu.